SGoopas/Assets/Scripts/Game/2D/2D Player States/AirDashRight2D.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace PlayerStates
+{
+    public class AirDashRight2D : Base2DState
```

Comment:
Shouldn't this inherit from BaseDashingState? Otherwise what's the point of BaseDashingState?

Reply:
Once upon a time i waa gonna make a base dashing state. Then i forgot about it. Then i finished and was like "oh boy. I wish i had made a base dashing state!" I forgot i actually made an empty class o.o

Reply:
BaseDashingState has been deleted